tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -422,3 +423,14 @@ Then('user {string} should have a share with these details:', function (user, ex
 Given('user {string} has created a new public link for resource {string}', function (user, resource) {
   return shareFileFolder(resource, user, '', SHARE_TYPES.public_link)
 })
+
+Given('the setting {string} in the {string} section has been disabled', function (setting, app) {
+  return occHelper.runOcc(
+    [
+      'config:app:set', app, setting, '--value=no'
+    ])
+})
+
+Then('the file/folder/resource {string} on shared-with-me page should be in pending state', function (filename) {
```

Comment:
if you mention the "shared-with-me" page, then you should also navigate there

Reply:
i'd navigated to shared-with-me page before this step.
while using pre-defined step it seemed odd